styles.css

```diff
@@ -128,23 +175,1194 @@ ul {
+  order: 1;
+  flex-grow: 0;
+  margin: auto;
+}
+
+.card_Work_Row_1 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 4px 2px;
+  gap: 24px;
+  width: 331px;
+  height: 2972px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.card {
+  width: 327px;
+  height: 474px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.text {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 16px;
+  position: absolute;
+
+  /* width: 327px;
+  height: 222px; */
+  left: calc (50% - 327px / 2);
+  bottom: 0;
+  background: #fff;
+  border-radius: 8px;
+}
+
+.project_title {
+  width: 295px;
+  height: 57px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #3a4a42;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.tags {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 8px;
+
+  /* width: 305px; */
+  height: 32px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+/* tag {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 8px 12px;
+  width: 103px;
+  height: 32px;
+  background: #EBF0EE;
+  border-radius: 4PX;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+} */
+.tag1 {
+  width: 79px;
+  height: 16px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #3a4a42;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.tag2 {
+  width: 21px;
+  height: 16px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #3a4a42;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.tag3 {
+  width: 58px;
+  height: 16px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #3a4a42;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.tag4 {
+  width: 27px;
+  height: 16px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #3a4a42;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.img_placeholder {
+  width: 327px;
+  height: 474px;
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  background: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+}
+
+.action1 {
+  /* width: 305px; */
+  height: 69px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
+
+.button {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  width: 122px;
+  height: 48px;
+  left: calc (50% - 122px / 2 + 0.5px);
+  bottom: 8px;
+  background: #36b37f;
+  border-radius: 4px;
+  border-style: none;
+}
```

Comment:
- [ ] As stated in the project requirement you are required to Implement the button interactions (enable, hover, etc.). So Kindly work on it.
![Capture](https://user-images.githubusercontent.com/93598189/205459171-2121b79b-11a8-4013-89f4-a55aafc0cae5.PNG)

The button button interactions from the figma design : 

![Capture](https://user-images.githubusercontent.com/93598189/205459235-ba4042ce-8b02-4cb5-b3d6-919d14f1e577.PNG)